tests/landing_page/landing_page.js

```diff
@@ -0,0 +1,116 @@
+            assert.include(text, "Pick an interest", "Ensure Quantity label is present");
+        })
+
+        it("test case: check service-title", async () => {
+            const element = await driver.findElement(By.css("body"))
+            const text = await element.getText();
+            assert.include(text, "How we serve you", "Ensure you have the appropriate text for the service");
+        })
+
+        it("test case: check service title", async () => {
+            const element = await driver.findElement(By.css("body"))
+            const text = await element.getText();
+            assert.include(text, "Tell us where you are", "Ensure you have the appropriate text for the service");
+        })
+
+        it("test case: check service-title", async () => {
+            const element = await driver.findElement(By.css("body"))
+            const text = await element.getText();
+            assert.include(text, "Tell us what you want", "Ensure you have the appropriate text for the service");
+        })
+
+        it("test case: check service-title", async () => {
+            const element = await driver.findElement(By.css("body"))
+            const text = await element.getText();
+            assert.include(text, "We'll come running", "Ensure you have the appropriate text for the service");
+        })
+
+        it("test case: check footer", async () => {
+            const element = await driver.findElement(By.css("footer"))
+            const text = await element.getText();
+            assert.include(text, "Contact us", "Ensure you have the appropriate text as seen on the mock-up");
+        })
+
+        it("test case: check service-title", async () => {
+            const element = await driver.findElement(By.css("footer"))
+            const text = await element.getText();
+            assert.include(text, "Terms & Conditions . Privacy Policy", "Ensure the texts on the footer is properly formated");
+        })
+
+        it("test case: check service-title", async () => {
+            const element = await driver.findElement(By.css("footer"))
+            const text = await element.getText();
+            assert.include(text, "© Copyright 2021 FoodBag is a registered trademark", "Ensure you have the appropriate text as seen on the mock-up");
+        })
+
+        it('test case: assert input field is present', async () => {
+            const input = await driver.findElements(By.css('input'));
+            const size = input.length
+            assert.equal(size, 1, `Ensure you have the input field as seen in the mock-up`);
+        });
+
+        it('test case: assert placeholder', async () => {
+            const element = await driver.findElement(By.css('input')).getAttribute('placeholder');
+            assert.equal(element, "Enter your delivery address", `Ensure you have the placeholder as seen in the mock-up`);
+        });
+
+        it('test case: assert image amount is 2', async () => {
```

Comment:
Since we'd be giving them the social media icons. You might want to increase this.